jar/list_events.ml

```diff
@@ -22,7 +22,7 @@ open Printf
 let string_of_wiki_page_action = function
   | `Created -> "Created"
   | `Edited -> "Edited"
-  | `Unknown (cons, _json) -> "Unknown:"^cons
+  | `Unknown cons -> "Unknown:"^cons
```

Comment:
Enums like this one now have an `Unknown` case of type `string` rather than `string * json option`. This comes from the use of the new `<json open_enum>` feature.